data/beaconrestapi/src/integration-test/java/tech/pegasys/teku/beaconrestapi/AbstractDataBackedRestAPIIntegrationTest.java

```diff
@@ -459,6 +460,12 @@ protected JsonNode getResponseData(final Response response) throws IOException {
     return body.get("data");
   }
 
+  protected void checkEmptyBodyToRoute(final String route, final int expectedResponseCode)
+      throws IOException {
+    final Response response = post(route, OBJECT_MAPPER.writeValueAsString(""));
```

Comment:
Is this what you want? Or do you need `"{}"`?

Reply:
this is literally what all the tests that did call, i just call objectMapper instead and centralised it